app/src/androidTest/java/org/wikipedia/travel/database/DestinationDbHelperTest.java

```diff
@@ -0,0 +1,43 @@
+package org.wikipedia.travel.database;
+
+import org.junit.Test;
+import org.wikipedia.travel.trip.Trip;
+
+import java.util.List;
+
+import static junit.framework.Assert.assertNotNull;
+import static junit.framework.Assert.assertTrue;
+import static org.junit.Assert.assertEquals;
+
+public class DestinationDbHelperTest {
+
+    @Test
+    public void testCreateAndDeleteList() {

```

Comment:
As it is a unit test, perhaps create separate tests for list creation and deletion 
Also, given the comments about last release's unit tests, is it possible to test edge cases such as the following deletion of list that doesn't exist in the db?

Reply:
Separated the test into their own unit and added edge test cases.